include/can/core/messages.hpp

```diff
@@ -198,6 +198,9 @@ using MotorPositionRequest = Empty<MessageId::motor_position_request>;
 using UpdateMotorPositionEstimationRequest =
     Empty<MessageId::update_motor_position_estimation_request>;
 
+using UpdateGearMotorPositionEstimationRequest =
```

Comment:
I don't understand what we're using to update gear motor position estimation? I don't think we have encoders on the tip motors, right?